variables.tf

```diff
@@ -292,3 +292,9 @@ variable "instance_cbr_rules" {
   default     = []
   # Validation happens in the rule module
 }
+
+variable "skip_iam_authorization_policy" {
+  type        = bool
+  description = "Set to true to skip the creation of IAM authorization policy. WARNING: An authorization policy must be created before an encrypted bucket can be created"
```

Comment:
Actually I was suggesting that we include more information on exactly the type of auth policy that gets created. Or maybe we add that to the main readme? But basically somewhere we need to let the consumer know that the auth policy that is created will allow the COS instance created by this module reader access to the Key Protect instance that is passed in with var. existing_key_protect_instance_guid